### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by ianaya89**

### CODE REVIEW

File: components/person/PersonMetaTags.vue [hunk 1]
```diff
@@ -5,14 +5,17 @@
...
     },
 
     head () {
-      const personPhotos = this.person.photos && this.person.photos.length ?  this.person.photos[0].url : ''
+      const personPhotos = this.person.photos && this.person.photos.length
+        ? this.person.photos[0].url : ''
+
       return {
-        title: this.person.name,
+        title: `Ayudanos a encontrar a ${this.person.name}`,
```
Comment (ndelvalle) [inline components/person/PersonMetaTags.vue:17]: I think this is to long for the tab title

**Review by ndelvalle — APPROVED** [review]
LGTM 👍

File: components/person/PersonMetaTags.vue (hunk 1 above)
Comment (ianaya89) [reply components/person/PersonMetaTags.vue:17]: Is not bad a long title, is something normal. Is better for SEO and is more expressive. Using just the name seems to be a profile page, something like this is more meaningful 

screen shot 2017-04-20 at 4 42 44 am

Comment (ndelvalle) [reply components/person/PersonMetaTags.vue:17]: Makes sense 👍

**Pull request merged**